documentation/modules/ROOT/pages/characteristics.adoc

```diff
@@ -592,6 +592,12 @@ At least one of `samm-c:maxValue` or `samm-c:minValue` must be present in a Leng
 | `samm-c:minValue` | The minimum length. Must be given as `xsd:nonNegativeInteger`. | {nok}
 |===
 
+IMPORTANT: When using the `samm-c:minValue` parameter in Length Constraint, it is crucial to consider the following:
+* Creating a Model with a <<set-characteristic>>/<<sorted-set-characteristic>> based on a finite Base Characteristic (`https://eclipse-esmf.github.io/samm-specification/snapshot/characteristics.html#boolean-characteristic[Boolean]` or `https://eclipse-esmf.github.io/samm-specification/snapshot/characteristics.html#enumeration-characteristic[Enumeration]`)
```

Comment:
We should always use internal refs or xrefs instead of hard-coded URLs when linking inside the docs:

```suggestion
Creating a Model with a <<set-characteristic>>/<<sorted-set-characteristic>> based on a finite Base Characteristic (<<boolean-characteristic>> or <<enumeration-characteristic>>)
```